lib/portage/_emirrordist/FetchTask.py

```diff
@@ -551,6 +552,49 @@ def _fetch_copier_exit(self, copier):
 			self.returncode = 1
 			self.wait()
 
+	def _make_layout_links(self):
+		dist_path = None
+		success = True
+		for layout in self.config.layouts[1:]:
+			if dist_path is None:
+				dist_path = os.path.join(self.config.options.distfiles,
+						self.config.layouts[0].get_path(self.distfile))
+			link_path = os.path.join(self.config.options.distfiles,
+					layout.get_path(self.distfile))
+			ensure_dirs(os.path.dirname(link_path))
+
+			if self.config.options.symlinks:
+				try:
+					os.symlink(
+							os.path.relpath(dist_path,
+								os.path.dirname(link_path)),
+							link_path)
```

Comment:
For completeness, we probably should support replacing an existing hardlink or symlink here. I suppose we could extend the _hardlink_atomic method to support symlink creation.

Reply:
…aaand done.